README.md

```diff
@@ -57,6 +47,31 @@ interface CommonBlockProps {
   ) => Promise<any>;
 
   onNavigateToPath: (path: string) => void;
+
+  // returns a list of repos containing blocks
+  onRequestBlocksRepos: (params?: {
+    path?: string; // restrict to blocks that match the path (see `matches` in `blocks.config.json`)
+    searchTerm?: string; // restrict to blocks matching the search term
+    repoUrl?: string; // restrict to blocks in the repo
+    type?: "file" | "folder"; // restrict to blocks of this type
+  }) => Promise<BlocksRepo[]>;
+
+  // a React component that renders a block viewing a file or folder
+  BlockComponent: ({
+    block,
+    context,
+  }: {
+    // owner, repo, and id identifying the block
+    block: { owner: string; repo: string; id: string };
+
+    // the file or folder to view; omitted fields default to the current file or folder
+    context: Partial<{
```

Comment:
I wonder if we want to clarify that `context` is optional

Reply:
💯